cmd/shipperctl/cmd/apply.go

```diff
@@ -234,6 +254,104 @@ func createManagementServiceAccount(cmd *cobra.Command, configurator *configurat
 	return nil
 }
 
+func createValidatingWebhookSecret(cmd *cobra.Command, configurator *configurator.Cluster) error {
+	fmt.Printf("Checking if a secret already exists for the validating webhook in the %s namespace... ", shipperSystemNamespace)
```

Comment:
We have a mix of ```fmt.Printf()``` and ```cmd.Println``` calls, which looks a bit misleading. Is there a way to use a single interface?

Reply:
Very nice catch. Fixed that.